helmfile.d/stacks/tekton.yaml

```diff
@@ -8,79 +8,76 @@ templates:
     name: tekton-rbac
     namespace: kube-system
```

Comment:
Do you know if there's any particular reason as to why we don't install tekton-rbac in the tekton-piplines namespace?

Reply:
If I'm not mistaken I believe it was @aarnq's suggestion to have it in `kube-system`, do you have an answer for this?

Reply:
Yes, because then it'll be the same for all clusters, without us being required to create tekton namespaces on workload clusters.